.mailmap

```diff
@@ -0,0 +1,50 @@
+Adriana Rivera-Dompenciel <ariveradompenciel@uiowa.edu>
+Adriana Rivera-Dompenciel <ariveradompenciel@uiowa.edu> <34017586+ariveradompenciel@users.noreply.github.com>
+Anibal Sólon Heinsfeld <anibalsolon@gmail.com>
+Asier Erramuzpe <asier.erramuzpe@gmail.com>
+Christopher J. Markiewicz <effigies@gmail.com>
+Christopher J. Markiewicz <effigies@gmail.com> <effigies@bu.edu>
+Christopher J. Markiewicz <effigies@gmail.com> <markiewicz@stanford.edu>
+Daniel J. Lurie <danjlurie@gmail.com>
+Elizabeth DuPre <emd222@cornell.edu>
+Franklin Feingold <ffein@stanford.edu>
+Franklin Feingold <ffein@stanford.edu> <35307458+franklin-feingold@users.noreply.github.com>
+Franz Liem <franz.liem@gmail.com>
+Gabriel A. Devenyi <gdevenyi@gmail.com>
+Ilkay Isik <ilkayisik@gmail.com>
+James D. Kent <james-kent@uiowa.edu>
+James D. Kent <james-kent@uiowa.edu> Fred Mertz <mertzf@bargle.argle>
```

Comment:
I messed something up during the beginning of my github experience, this line should be
```suggestion
James D. Kent <james-kent@uiowa.edu> <jamesdkent21@gmail.com>
```

Reply:
@jdkent The point of this line is to attribute those commits to you. We can also add `jamesdkent21@gmail.com` if you like.

Reply:
oh, I see, that would be great!
I'm also fine with dropping Fred Mertz, since I do not believe I made any commits to fmriprep under that mistaken alias.

Reply:
```
fmriprep git:(master) ✗ git log | grep Fred
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
Author: Fred Mertz <mertzf@bargle.argle>
```

Reply:
I stand corrected, thanks!